### PR TITLE
URL documentation helper

### PR DESCRIPTION
**Pull request by hdonnay**
This adds and then uses a helper to extract URLs from go source directly into documentation.

### CODE REVIEW

**Comment by hdonnay** [comment]
Here's a the rendered documentation:

image

File: crda/remotematcher.go [hunk 1]
```diff
@@ -25,7 +25,8 @@ var (
 )
 
 const (
-	batchSize     = 10
+	batchSize = 10
+	//doc:url indexer
```
Comment (crozzy) [inline crda/remotematcher.go:29]: I would have thought this warranted it's own category

**Review by crozzy — COMMENTED** [review]
Nice idea,  just one question

**Review by crozzy — APPROVED** [review]

**Comment by hdonnay** [comment]
Merging with green CI

**Pull request merged**